test/rclone_export/docker-compose.yaml

```diff
@@ -34,9 +33,6 @@ services:
 
   lakefs-export:
     image: "${REPO:-treeverse}/lakefs-rclone-export:${EXPORT_TAG:-latest}"
-    depends_on:
-      - "lakefs"
-      - "postgres"
```

Comment:
Not blocking on this, sharing my thoughts though:
This solves the problem, and because we run it in later steps it works to overcome the dependencies, but we regarding the docker-compose file it's wrong to remove these dependencies. 
Did we try using project names?

Reply:
I understand the concern, however:
1. I feel like we should not invest more time on this since this provides a good enough solution
2. Lets address this when this actually becomes painful